service/src/main/java/uk/gov/hmcts/reform/fpl/model/HearingBooking.java

```diff
@@ -27,8 +28,7 @@
     private final LocalDateTime endDate;
     private final List<String> hearingNeedsBooked;
     private final String hearingNeedsDetails;
-    private final String judgeTitle;
-    private final String judgeName;
+    private final JudgeAndLegalAdvisor judgeAndLegalAdvisor;
```

Comment:
I am not fan of these `XandY` classes. They do too much, they have too many reasons to change. They are bad pattern, bad code smell.

Again not issue directly with this PR as you just apply existing solution in consistent manner.